javascript/javascript2/week3/task3.js

```diff
@@ -0,0 +1,88 @@
+
+
+
+const refs = {
+    input: document.querySelector('[data-input=value]'),
+    button: document.querySelector('.btn-start'),
+    outputS: document.querySelector('.outputS'),
+    outputL: document.querySelector('.outputL'),
+    timeOver: document.querySelector('.timer-over'),
+    divS: document.querySelector('.gamer-s'),
+    divL: document.querySelector('.gamer-l'),
+    
+}
+let inputValue = 0;
+let counterS = 0;
+let counterL = 0;
+
+refs.button.addEventListener('click', onClickBtnStart);
+document.addEventListener('keyup', onKeyupCounter)
+
+
+
+function onClickBtnStart(event) {
+    inputValue = refs.input.value;
+    if (inputValue === "") {
+       return refs.timeOver.textContent = 'Set timeout'
+    }
+    const timeoutId = setTimeout( defineWinner,
+        inputValue * 1000)
+        
+        if (inputValue !== "") {
+         return refs.timeOver.textContent = ''
+     }
+}
+
+function defineWinner() {
+       if (counterL === counterS) {
+           alert('A draw')
```

Comment:
Displaying this message somewhere on the screen instead of using alert would give a better user experience to your game :)
In general try to avoid alert as much as possible.

Reply:
Ok.) Thank you ! 
I didn't think about it, yes, I will try to avoid alerts.